test-data/unit/check-possibly-undefined.test

```diff
@@ -909,6 +909,16 @@ def f0() -> None:
     type = "abc"
     a = type
 
+[case testUsedBeforeDefBuiltinsMultipass]
+# flags: --enable-error-code used-before-def
+
+# When doing multiple passes, mypy resolves references slightly differently.
+# In this case, it would refer the earlier `range` call to the range class defined below.
```

Comment:
Change `range` in the comment to `type`.

Reply:
Oops! I realized that `range` wasn't included in fixtures by default, so switched to `type` but didn't update the comment. Thanks!